### ISSUE TITLE
IOS Project Build Fails when using NMSSH...

### ISSUE THREAD

**Issue by darryljackman** [Lejdborg/NMSSH#119]
If I use:

NMSSHSession *session = [NMSSHSession connectToHost:@"127.0.0.1:22"
                                           withUsername:@"user"];

I get the following error:

Undefined symbols for architecture armv7:
  "_OBJC_CLASS_$_NMSSHSession", referenced from:
      objc-class-ref in ViewController.o
ld: symbol(s) not found for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by bhr** [comment]
Looks like NMSSH framework cannot be found. How do you include NMSSH into your project? CocoaPods or manually?

If manually, look if the framework search paths are set correctly under `Build Settings` in your target.

Another thing to check is if the framework has been copied correctly to the right place. Therefore, go to Products Group in Xcode -> Ctrl-Click on <#MyApp#>.app -> Show in Finder -> Ctrl-Click on the file <#MyApp#>.app. There you should see `NMSSH.framework`. Inside the framework should be a folder `Headers` and the compiled framework `NMSSH`. If not, please report.

What helps in general is to take a look at "Build details" tab in Xcode. It's the right tab in the left sidebar. There, you can reveal all warnings (if a file has not been found) and errors in detail.

**Comment by darryljackman** [comment]
I didn't get a chance to look where you suggested as I had already found a way to get it to work, but in case it helps, this happen when I tried to use the 'pod' instead of manually.

This could also just be me, but I've used 5-6 different 'pods' in projects before, and i do the usual add the line to the podfile, then 'pod install', and magically its there. Then I opened my Workspace again as normal, and was able to #import the <NMSSH/NMSSH.h> without error, and i could even put in the example code from the main page, and the compiler showed no errors, and all the class names and methods and stuff were coloured correctly, so it was finding the necessary files/class up to this point. It was when I tried to compile that the error came up. Other pods i've used before i could just start using after the 'pod install' finished, so don't know what i was missing here.

After several attempts of of uninstalling and re-installing the pod, it was always the same, and I read somewhere that the framework had to be built for the right platform, but i installed this through the 'pod', and would have no idea how to re-build a framework for a different platform.

Great library though, I am already excited about the abilities it will give to my app, which involves sending various commands to an embedded device on the local network.

Also excuse me, I am new to github, but when is the best time to close an issue? I have gotten past my specific issue, and maybe no one else even had this problem, so how to know for sure if is an issue or non-issue?? My apologies..
